docs/announcements/github-release-v0.1.74.md

```
@@ -0,0 +1,23 @@
+# 🚀 Release Highlights — v0.1.74 (2026-04-08)
+
+### 🛡️ [Checkpoint MCP Improvements](https://github.com/massgen/MassGen/blob/main/docs/modules/checkpoint.md)
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)